### PR TITLE
Adding eigenpy to documentation index for indigo

### PR DESCRIPTION
**Pull request by nim65s**
Eigenpy gives bindings between numpy and eigen using boost::python.

### CODE REVIEW

File: indigo/distribution.yaml [hunk 1]
```diff
@@ -2901,6 +2901,16 @@ repositories:
...
       version: master
     status: maintained
+  eigenpy:
+    doc:
+      type: git
+      url: https://github.com/stack-of-tasks/eigenpy.git
+      version: master
+    source:
+      type: git
+      url: https://github.com/nim65s/eigenpy.git
```
Comment (mikaelarguedas) [inline indigo/distribution.yaml]: Is it on purpose that the documentation comes from a repo and the source code comes from a fork of that repo?

Comment (nim65s) [reply indigo/distribution.yaml]: Yes, I need some time to understand the ROS ecosystem and to adapt our code for catkin & bloom.
So if possible I'd like to use my fork as a starting point while it's not fully operationnal, before pushing the modifications to the upstream.

But I guess I should also use my fork for the doc, as the upstream doesn't have a `package.xml` file for now.

**Review by mikaelarguedas — APPROVED** [review]
sounds good, thanks for iterating

**Comment by nim65s** [comment]
Thanks :)

**Pull request merged**